styles.css

```diff
@@ -0,0 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Domine&family=Montserrat:wght@100&display=swap');
+
+body {
+  background-color: #6f23ff;
+  font-family: 'Montserrat', sans-serif;
+  margin: 50px 100px 50px 100px;
+  color: white;
+  text-align: center; 
+  background-repeat: re;
+}
```

Comment:
Good thing implementing the background for your website. You can check on the css properties for background-repeat then use a valid property

Reply:
- [ ] Great work making this comment. It was polite, clear and easy to understand. However, you did not give reasons why the author should make changes to the highlighted code. Clear and specific reasons help the author of the code to understand the intent behind the suggested changes, fostering better communication and reducing ambiguity. This allows for a more constructive dialogue between team members. 👍 

#### Hint: Edit your comment above and give reasons why there is a need to remove unused code in the codebase. 😎